api/types/resource.go

```diff
@@ -90,14 +90,21 @@ type ResourceWithLabels interface {
 // ResourcesWithLabels is a list of labeled resources.
 type ResourcesWithLabels []ResourceWithLabels
```

Comment:
Do we still need this type?

Reply:
Yes, we do: it is rather popular in the codebase. I could work to replace it, but not everywhere `ResourcesWithLabelsMap` makes sense.